Adyen/Model/Checkout/PaymentMethod.cs

```diff
@@ -47,7 +47,7 @@ public partial class PaymentMethod :  IEquatable<PaymentMethod>, IValidatableObj
         /// <param name="PaymentMethodData">Echo data required to send in next calls..</param>
         /// <param name="SupportsRecurring">Indicates whether this payment method supports tokenization or not..</param>
         /// <param name="Type">The unique payment method code..</param>
```

Comment:
```suggestion
        /// <param name="Type">The unique payment method code..</param>
        /// <param name="Brands">List of possible brands. For example: visa, mc.</param>
```
or something like that